Updated readme because of new functionality (#79)
thepieterdc/dodona-plugin-vscode: README.md

1. <img src="assets/sideview.png" alt="cogwheel" width=30px;/> Dodona icon in the Activity Bar (icon in left column)
2. `Open Settings`
   ![Open settings](assets/open-settings.png)
3. Paste the API token you just made in text box.

> 4. In the left column you get an overview of the exercise series of the course (expand it if necessary).

Author: Can you change this to ` In the left column you get an overview of the exercises and series of the course (expand it if necessary).`?

Reviewer: Like this?
```suggestion
4. In the left column you get an overview of the exercises and series of the course (expand it if necessary).
```

Author: Yes, thank you.

| :--------------------------------------------------------------------: | :------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
|       <img src="assets/refresh.svg" alt="refresh" width=15px;/>        | Reload the list of activities.                                                                                                                                                        |
|  <img src="assets/notification.svg" alt="notification" width=15px;/>   | Notifies users in case of unread feedback. The icon links to https://dodona.ugent.be/notifications.                                                                                   |
| <img src="assets/content-page-read.svg" alt="checkmark" width=15px;/>  | Submit solution to Dodona. Alternative for `Alt+D` and `Ctrl+Shift+P`, `Dodona: Submit Solution`.                                                                                     |

> |          <img src="assets/edit.svg" alt="edit" width=15px;/>           | Opens the description of an exercise series or course in a new browser window. (Hover with your cursor above an exercise series or course in the left column with Dodona activities.) |

Reviewer: Use commas when summing up, hover `over`
```suggestion
|          <img src="assets/edit.svg" alt="edit" width=15px;/>           | Opens the description of an exercise, series or course in a new browser window. (Hover with your cursor over an exercise, series or course in the left column with Dodona activities.) |
```

Author: I meant "oefeningenreeks" in Dutch. Does this translate to `exercise series`?

Reviewer: Oh, okay. I'll edit my suggestion.

In that case: both Dodona itself and our plugin refer to these as `series` everywhere, so for the sake of consistency I feel like it's better to stick with that. If you look at the URLs you'll also notice this.

`dodona.ugent.be/series/XXX`, `dodona.ugent.be/courses/XXX/series`, etc.

Author: I searched for it here https://dodona-edu.github.io/en/guides/teachers/exercise-series-management/ but there was no English translation available of that page. Thank you for your clarification.

> | <img src="assets/open-description.svg" alt="description" width=15px;/> | Show the exercise description in VS Code. (Hover with your cursor above an exercise in the left column with Dodona activities.)                                                       |

Reviewer: ```suggestion
| <img src="assets/open-description.svg" alt="description" width=15px;/> | Show the exercise description in VS Code. (Hover with your cursor over an exercise in the left column with Dodona activities.)                                                       |
```
